sdk/cosmos/azure-cosmos/samples/access_cosmos_with_aad_async.py

```diff
@@ -0,0 +1,111 @@
+from azure.cosmos.aio import CosmosClient
```

Comment:
Missing license. 
We should also fix the existing samples and tests if the licenses are missing. 
Also, wondering why doesn't the azure engineering tooling catches this? 
cc @annatisch  - we do this on java SDKs, wondering if there is anything for this on python SDK ?

Reply:
That's actually a good question to which I don't have a good answer.
Let me raise it with the team and see if we can get a CI check added for that :)

Reply:
Just realized none of our samples have licenses up top, will add them now